pkg/chartmuseum/server/multitenant/server.go

```diff
@@ -187,3 +200,21 @@ func (server *MultiTenantServer) genIndex() {
 	echo(string(entry.RepoIndex.Raw[:]))
 	exit(0)
 }
+
+func (server *MultiTenantServer) CheckTemplateFilesExist(path string, logger *cm_logger.Logger) bool {
+	// check if template file exists
+	webTemplateFolder, err := os.Open(path)
+	if err != nil {
```

Comment:
log error here to report whether if check failed is due to some filesystem issue is better ?

Reply:
Done, added error log.